Makefile

```diff
@@ -517,7 +517,7 @@ bench-idle:
 	$(NODE) benchmark/idle_clients.js &
 
 jslint:
-	$(NODE) tools/eslint/bin/eslint.js src lib test tools/eslint-rules \
+	$(NODE) tools/eslint/bin/eslint.js lib src test tools/doc tools/eslint-rules \
```

Comment:
Ah, yes, of course! Done! (If someone on a Windows machine can confirm that it still works on Windows, that would be great. I don't think we run lint on Windows in CI.)